main/run_multiple_sim.jl

```diff
@@ -109,21 +108,29 @@ function run_multiple_sim(N=1;
         x0s = 1:N |> Map(i -> FTCTests.sample(multicopter, distribution_info(manoeuvre)...)) |> collect
         for method in [:adaptive, :adaptive2optim]
             dir_log = joinpath(joinpath(_dir_log, String(manoeuvre)), String(method))
-            case_numbers_partition = 1:N |> Partition(N_thread; flush=true) |> Map(copy) |> collect
-            for case_numbers in case_numbers_partition
-                @time _ = zip(case_numbers,
-                            x0s[case_numbers],
-                            _faults[case_numbers],
-                            τs[case_numbers]) |>
-                MapSplat((i, x0, _fault, τ) -> FTCTests.run_sim(method, x0, multicopter,
-                                                                FaultSet(_fault...),
-                                                                DelayFDI(τ),
-                                                                traj_des, dir_log, i;
-                                                                will_plot=will_plot,
-                                                                t0=t0, tf=tf,
-                                                                h_threshold=h_threshold,
-                                                                actual_time_limit=actual_time_limit,
-                                                            )) |> collector
+            if method == :adaptive
+                τs = [0.0]
+            elseif method == :adaptive2optim
+                τs = [0.0, 0.1]
+            else
+                error("Invalid method")
+            end
+            for τ in τs
```

Comment:
```suggestion
            for (j, τ) in enumerate(τs)
```

여기는 optional; 기존의 디렉토리명이 바뀌는게 불편하면 이것은 어떨까.. 정도로?